### PR TITLE
docs: update VS Code getting started for Uno 5.2

### PR DESCRIPTION
**Pull request by spouliot**
GitHub Issue (If applicable): closes #

## PR Type

What kind of change does this PR introduce?

- Documentation content changes

## What is the current behavior?

Existing VS Code documentation won't match with the upcoming changes in Uno 5.2.

## What is the new behavior?

This update the getting started with the Uno 5.2 changes and a few others (like WinUI support) from the previous release of the extension.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Comment by spouliot** [comment]
note: should not be merged before 5.2 is ready

File: doc/articles/create-an-app-vscode.md [hunk 1]
```diff
@@ -41,23 +38,47 @@ Next, open the project using Visual Studio Code:
     ```
 
 * Visual Studio Code might ask to restore the NuGet packages. Allow it to restore them if asked.
-* Once the project has been loaded, in the status bar at the bottom left of VS Code, `MyApp.sln` is selected by default. Select `MyApp.Wasm.csproj`, `MyApp.Skia.Gtk.csproj` or `MyApp.Mobile.csproj` instead.
+* Once the solution has been loaded, in the status bar at the bottom left of VS Code, `MyApp.sln` is selected by default. Select `MyApp.csproj` to load the project instead.
 
 ## Debug the App
 
-Uno Platform provides integrated support for debugging your app on Windows, mac and Linux.
+Uno Platform provides integrated support for debugging your app on Windows, macOS and Linux.
```
Comment (agneszitte) [inline doc/articles/create-an-app-vscode.md:45]:
```suggestion
Uno Platform provides integrated support for debugging your app on Windows, macOS, and Linux.
```

File: doc/articles/create-an-app-vscode.md [hunk 2]
```diff
@@ -114,14 +145,44 @@ Note that C# Hot Reload is not available when running with the debugger. In orde
...
+1. In the [activity bar](https://code.visualstudio.com/docs/getstarted/userinterface) select the debugger
+1. Set the debugger launch to `Uno Platform Mobile`
+
+![VS Code UI](Assets/quick-start/vs-code-mobile-debug.png)
+
+In the status bar
+
+1. Just after the Uno logo ensure the `MyApp` project is selected - by default `MyApp.sln` is selected
+1. Next click on the target framework to select `net8.0-windows10.0.xxxxx | Debug`
+1. The `This Computer` device will be pre-selected. On ARM64-based computers you will have the option to use `This Computer using emulation` to debug `x64` applications
```
Comment (agneszitte) [inline doc/articles/create-an-app-vscode.md:181]:
```suggestion
1. The `This Computer` device will be pre-selected. On ARM64-based computers, you will have the option to use `This Computer using emulation` to debug `x64` applications
```

**Review by agneszitte — COMMENTED** [review]
@jeromelaban, @spouliot 
By testing the changes for this PR, I have noticed an issue for the code snippet sections where they are not displaying well when apparently they are used in tabs.
I think I will open a separate issue for it, as it is already like that in prod, but wanted to have your feedback on it first.
image

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/5.2

File: doc/articles/create-an-app-vscode.md [hunk 3]
```diff
@@ -41,23 +38,47 @@ Next, open the project using Visual Studio Code:
...
 
-Uno Platform provides integrated support for debugging your app on Windows, mac and Linux.
+Uno Platform provides integrated support for debugging your app on Windows, macOS and Linux.
 
 ### [**WebAssembly**](#tab/Wasm)
 
-1. In the debugger section of the [activity bar](https://code.visualstudio.com/docs/getstarted/userinterface) select `Debug (Chrome, WebAssembly)`
-1. In the status bar, ensure the `MyApp.Wasm.csproj` project is selected - by default `MyApp.sln` is selected.
-1. Press `F5` to start the debugging session
+In VS Code
```
Comment (jeromelaban) [inline doc/articles/create-an-app-vscode.md:49]:
```suggestion
In VS Code :
```

File: doc/articles/create-an-app-vscode.md [hunk 4]
```diff
@@ -41,23 +38,47 @@ Next, open the project using Visual Studio Code:
...
+
+1. In the [activity bar](https://code.visualstudio.com/docs/getstarted/userinterface) select the debugger
+1. Set the debugger launch to `Debug (Chrome, WebAssembly)`
+
+![VS Code UI](Assets/quick-start/vs-code-chrome-debug.png)
+
+In the status bar
```
Comment (jeromelaban) [inline doc/articles/create-an-app-vscode.md:56]:
```suggestion
In the status bar :
```

File: doc/articles/create-an-app-vscode.md [hunk 5]
```diff
@@ -41,23 +38,47 @@ Next, open the project using Visual Studio Code:
...
+
+1. Just after the Uno logo ensure the `MyApp` project is selected - by default `MyApp.sln` is selected
```
Comment (jeromelaban) [inline doc/articles/create-an-app-vscode.md:58]:
```suggestion
1. Just after the Uno logo, ensure the `MyApp` project is selected - by default `MyApp.sln` is selected
```

File: doc/articles/create-an-app-vscode.md [hunk 6]
```diff
@@ -41,23 +38,47 @@ Next, open the project using Visual Studio Code:
...
+1. Next click on the target framework to select `net8.0-browserwasm | Debug`
```
Comment (jeromelaban) [inline doc/articles/create-an-app-vscode.md:59]:
```suggestion
1. Next, click on the target framework to select `net8.0-browserwasm | Debug`
```

File: doc/articles/create-an-app-vscode.md [hunk 7]
```diff
@@ -41,23 +38,47 @@ Next, open the project using Visual Studio Code:
...
+
+![status bar](Assets/quick-start/vs-code-browserwasm-project.png)
+
+Finally press `F5` to start the debugging session.
```
Comment (jeromelaban) [inline doc/articles/create-an-app-vscode.md:63]:
```suggestion
Finally, press `F5` to start the debugging session.
```

File: doc/articles/create-an-app-vscode.md [hunk 8]
```diff
@@ -41,23 +38,47 @@ Next, open the project using Visual Studio Code:
...
+
+### [**Desktop / Skia**](#tab/skia)
+
+In VS Code
```
Comment (jeromelaban) [inline doc/articles/create-an-app-vscode.md:67]:
```suggestion
In VS Code :
```

File: doc/articles/create-an-app-vscode.md [hunk 9]
```diff
@@ -41,23 +38,47 @@ Next, open the project using Visual Studio Code:
...
-### [**Skia GTK**](#tab/skiagtk)
+1. In the [activity bar](https://code.visualstudio.com/docs/getstarted/userinterface) select the debugger
+1. Set the debugger launch to `Uno Platform Mobile` or `Uno Platform Desktop (Debug)`
 
-1. In the debugger section of the [activity bar](https://code.visualstudio.com/docs/getstarted/userinterface) select `Skia.GTK (Debug)`
-1. In the status bar, ensure the `MyApp.Skia.Gtk.csproj` project is selected - by default `MyApp.sln` is selected.
-1. Press `F5` to start the debugging session
+![VS Code UI](Assets/quick-start/vs-code-mobile-debug.png)
+
+In the status bar
```
Comment (jeromelaban) [inline doc/articles/create-an-app-vscode.md:74]:
```suggestion
In the status bar :
```

File: doc/articles/create-an-app-vscode.md [hunk 10]
```diff
@@ -41,23 +38,47 @@ Next, open the project using Visual Studio Code:
...
+
+1. Just after the Uno logo ensure the `MyApp` project is selected - by default `MyApp.sln` is selected
```
Comment (jeromelaban) [inline doc/articles/create-an-app-vscode.md:76]:
```suggestion
1. Just after the Uno logo, ensure the `MyApp` project is selected - by default `MyApp.sln` is selected
```

File: doc/articles/create-an-app-vscode.md [hunk 11]
```diff
@@ -41,23 +38,47 @@ Next, open the project using Visual Studio Code:
...
+1. Next click on the target framework to select `net8.0-desktop | Debug`
+
+![status bar](Assets/quick-start/vs-code-desktop-project.png)
+
+Finally press `F5` to start the debugging session.
```
Comment (jeromelaban) [inline doc/articles/create-an-app-vscode.md:81]:
```suggestion
Finally, press `F5` to start the debugging session.
```

File: doc/articles/create-an-app-vscode.md [hunk 12]
```diff
@@ -77,34 +98,44 @@ Note that C# Hot Reload is not available when running with the debugger. In orde
 
 ### [**Android**](#tab/androiddebug)
 
-* In the status bar, select the `MyApp.Mobile` project - by default `MyApp.sln` is selected.
+In VS Code
```
Comment (jeromelaban) [inline doc/articles/create-an-app-vscode.md:101]:
```suggestion
In VS Code :
```

File: doc/articles/create-an-app-vscode.md [hunk 13]
```diff
@@ -77,34 +98,44 @@ Note that C# Hot Reload is not available when running with the debugger. In orde
...
+1. Set the debugger launch to `Uno Platform Mobile`
 
-  ![android target framework](Assets/quick-start/vs-code-debug-tf-android.png)
-* Then, to the right of the target framework, select the device to debug with. You will need to connect an android device, or create an Android simulator.
+![VS Code UI](Assets/quick-start/vs-code-mobile-debug.png)
 
-  ![android device name](Assets/quick-start/vs-code-debug-device-android.png)
-* Finally, in the debugger side menu, select the `Uno Platform Mobile` profile
-* Either press `F5` or press the green arrow to start the debugging session.
+In the status bar
```
Comment (jeromelaban) [inline doc/articles/create-an-app-vscode.md:108]:
```suggestion
In the status bar :
```

File: doc/articles/create-an-app-vscode.md [hunk 14]
```diff
@@ -77,34 +98,44 @@ Note that C# Hot Reload is not available when running with the debugger. In orde
...
+
+1. Just after the Uno logo ensure the `MyApp` project is selected - by default `MyApp.sln` is selected
+1. Next click on the target framework to select `net8.0-android | Debug`
+1. Then select the device to debug with. You will need to connect an Android device or [create an Android emulator](https://developer.android.com/studio/run/managing-avds).
+
+![status bar](Assets/quick-start/vs-code-android-project.png)
+
+Finally press `F5` to start the debugging session.
```
Comment (jeromelaban) [inline doc/articles/create-an-app-vscode.md:116]:
```suggestion
Finally, press `F5` to start the debugging session.
```

File: doc/articles/create-an-app-vscode.md [hunk 15]
```diff
@@ -77,34 +98,44 @@ Note that C# Hot Reload is not available when running with the debugger. In orde
...
 
 ### [**iOS**](#tab/iosdebug)
 
 > [!NOTE]
 > Debugging for iOS is only possible when running locally (or remotely through [Remote SSH](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-ssh)) on a macOS machine.
 
-* In the status bar, select the `MyApp.Mobile` project - by default `MyApp.sln` is selected.
+In VS Code
```
Comment (jeromelaban) [inline doc/articles/create-an-app-vscode.md:123]:
```suggestion
In VS Code :
```

File: doc/articles/create-an-app-vscode.md [hunk 16]
```diff
@@ -77,34 +98,44 @@ Note that C# Hot Reload is not available when running with the debugger. In orde
...
+1. In the [activity bar](https://code.visualstudio.com/docs/getstarted/userinterface) select the debugger
+1. Set the debugger launch to `Uno Platform Mobile`
 
-  ![mobile project name](Assets/quick-start/vs-code-debug-project.png)
-* To the right of `MyApp.Mobile`, click on the target framework to select `net8.0-ios | Debug`
+![VS Code UI](Assets/quick-start/vs-code-mobile-debug.png)
 
-  ![ios target framework](Assets/quick-start/vs-code-debug-tf-ios.png)
-* Then, to the right of the target framework, select the device to debug with. You will need to connect an iOS device, or use an existing iOS simulator.
+In the status bar
```
Comment (jeromelaban) [inline doc/articles/create-an-app-vscode.md:130]:
```suggestion
In the status bar :
```

File: doc/articles/create-an-app-vscode.md [hunk 17]
```diff
@@ -77,34 +98,44 @@ Note that C# Hot Reload is not available when running with the debugger. In orde
...
-  ![ios device](Assets/quick-start/vs-code-debug-device-ios.png)
-* Finally, in the debugger side menu, select the `Uno Platform Mobile` profile
-* Either press `F5` or press the green arrow
+1. Just after the Uno logo ensure the `MyApp` project is selected - by default `MyApp.sln` is selected
+1. Next click on the target framework to select `net8.0-ios | Debug`
+1. Then select the device to debug with. You will need to connect an iOS device or [use an installed iOS simulator](https://developer.apple.com/documentation/xcode/installing-additional-simulator-runtimes).
+
+![status bar](Assets/quick-start/vs-code-ios-project.png)
+
+Finally press `F5` to start the debugging session.
```
Comment (jeromelaban) [inline doc/articles/create-an-app-vscode.md:138]:
```suggestion
Finally, press `F5` to start the debugging session.
```

File: doc/articles/create-an-app-vscode.md [hunk 18]
```diff
@@ -114,14 +145,44 @@ Note that C# Hot Reload is not available when running with the debugger. In orde
...
 
-* In the status bar, select the `MyApp.Mobile` project - by default `MyApp.sln` is selected.
+In VS Code
+
+1. In the [activity bar](https://code.visualstudio.com/docs/getstarted/userinterface) select the debugger
+1. Set the debugger launch to `Uno Platform Mobile`
+
+![VS Code UI](Assets/quick-start/vs-code-mobile-debug.png)
+
+In the status bar
```
Comment (jeromelaban) [inline doc/articles/create-an-app-vscode.md:155]:
```suggestion
In the status bar :
```

File: doc/articles/create-an-app-vscode.md [hunk 19]
```diff
@@ -114,14 +145,44 @@ Note that C# Hot Reload is not available when running with the debugger. In orde
 > [!NOTE]
 > Debugging for Mac Catalyst is only possible when running locally (or remotely through [Remote SSH](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-ssh)) on a macOS machine.
 
-* In the status bar, select the `MyApp.Mobile` project - by default `MyApp.sln` is selected.
+In VS Code
```
Comment (jeromelaban) [inline doc/articles/create-an-app-vscode.md:148]:
```suggestion
In VS Code :
```

File: doc/articles/create-an-app-vscode.md [hunk 20]
```diff
@@ -114,14 +145,44 @@ Note that C# Hot Reload is not available when running with the debugger. In orde
...
+
+In the status bar
+
+1. Just after the Uno logo ensure the `MyApp` project is selected - by default `MyApp.sln` is selected
+1. Next click on the target framework to select `net8.0-maccatalyst | Debug`
+1. The `This Mac` device will be pre-selected. On Apple Silicon (arm64) Macs you will have the option to use `This Mac using Rosetta` to debug `x64` applications
+
+![status bar](Assets/quick-start/vs-code-maccatalyst-project.png)
+
+Finally press `F5` to start the debugging session.
```
Comment (jeromelaban) [inline doc/articles/create-an-app-vscode.md:163]:
```suggestion
Finally, press `F5` to start the debugging session.
```

File: doc/articles/create-an-app-vscode.md [hunk 21]
```diff
@@ -114,14 +145,44 @@ Note that C# Hot Reload is not available when running with the debugger. In orde
...
+
+### [**Windows**](#tab/windowsdebug)
+
+> [!NOTE]
+> Debugging for Windows is only possible when running locally (or remotely through [Remote SSH](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-ssh)) on a Windows computer.
+
+In VS Code
```
Comment (jeromelaban) [inline doc/articles/create-an-app-vscode.md:170]:
```suggestion
In VS Code :
```

File: doc/articles/create-an-app-vscode.md [hunk 22]
```diff
@@ -114,14 +145,44 @@ Note that C# Hot Reload is not available when running with the debugger. In orde
...
+
+1. In the [activity bar](https://code.visualstudio.com/docs/getstarted/userinterface) select the debugger
+1. Set the debugger launch to `Uno Platform Mobile`
+
+![VS Code UI](Assets/quick-start/vs-code-mobile-debug.png)
+
+In the status bar
```
Comment (jeromelaban) [inline doc/articles/create-an-app-vscode.md:177]:
```suggestion
In the status bar :
```

**Comment by agneszitte** [comment]
> @jeromelaban, @spouliot By testing the changes for this PR, I have noticed an issue for the code snippet sections where they are not displaying well when apparently they are used in tabs. I think I will open a separate issue for it, as it is already like that in prod, but wanted to have your feedback on it first. 

I have opened a separate issue for it: https://github.com/unoplatform/uno/issues/16100

**Pull request merged**